src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java

```diff
@@ -77,14 +77,10 @@
  * <p>
  * Examples of using Unicode:</p>
  * <pre>
- * String unitAbbrev = "μs";      // Best: perfectly clear even without a comment.
- * String unitAbbrev = "&#92;u03bcs"; // Poor: the reader has no idea what this is.
- * </pre>
- * <p>
- * An example of non-printable, control characters.
- * </p>
- * <pre>
- * return '&#92;ufeff' + content; // byte order mark
+ * String unitAbbrev = "μs"; // OK, perfectly clear even without a comment.
+ * String unitAbbrev = "&#92;u03bcs"; // violation, the reader has no idea what this is.
+ * return '&#92;ufeff' + content; // OK, an example of non-printable,
+ *                               // control characters (byte order mark).
```

Comment:
look at https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/AvoidEscapedUnicodeCharacters_2021045318/config_misc.html#AvoidEscapedUnicodeCharacters_Examples

trailing comment are hard to read because they are not vertically aligned.
can we make it looks like:
```
 * String unitAbbrev = "μs";          // OK, perfectly clear even without a comment.
 * String unitAbbrev = "&#92;u03bcs"; // violation, the reader has no idea what this is.
```
it is ok to not todo this for lines that are too long and exceed limit of LineLength Check.
please apply this for all examples you provide.

Reply:
done